ui/helpers/utils/util.js

```diff
@@ -411,6 +411,20 @@ export function checkExistingAddresses(address, list = []) {
   return list.some(matchesAddress);
 }
 
+export function checkExistingAllTokens(
+  address,
+  chainId,
+  accountAddress,
+  list = {},
+) {
+  if (!address) {
+    return false;
+  }
+  return list?.[chainId]?.[accountAddress]?.some(
+    (obj) => obj.address.toLowerCase() === address.toLowerCase(),
+  );
+}
+
```

Comment:
It would be nice if this function could be introduced into a typescript file. Is there an easy way to do this?

Additionally, can we also add a unit test in `token-util.test..js`?